book/website/reproducible-research/reviewing/reviewing-workflow.md

```diff
@@ -48,6 +48,14 @@ Once you believe changes are complete, the reviewer checks that they do indeed a
 
 Once post-review changes have been made to the code, make final updates the comments as needed to complete a history of what has been done and the reasoning behind it.
 
+## Build and Run the Code
+
+Code review isn't just about passive review. The majority of pull requests will contain elements that will be hard to understand through inspection of the text documents alone.
```

Comment:
"will be hard" -> "could be difficult"
(sounds less daunting)

Are "text documents" code files?